services/api/src/migrations/20211116140144_payment_bank_account.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('PaymentBankAccount', (table) => {
+    table.uuid('id').primary()
+    table.uuid('externalId').notNullable().unique()
+    table.integer('amount').notNullable()
+    table.text('status').notNullable()
+    table.text('description').notNullable()
+    table.uuid('ownerId').notNullable().references('id').inTable('UserAccount')
+    table.timestamp('createdAt').defaultTo(knex.fn.now()).notNullable()
+    table.timestamp('updatedAt').defaultTo(knex.fn.now()).notNullable()
+  })
+  await knex.schema.alterTable('Payment', (table) => {
+    table.uuid('paymentBankId').references('id').inTable('PaymentBankAccount')
+    table.uuid('externalId').nullable().alter()
+  })
+}
```

Comment:
The `Payment` table needs to allow null values for the external ID, so that a payment can be created for the wire transfer before the payment is available via Circle's API. This links the payment as well as the pack to the wire transfer. When the scheduler function is added, we can update the payment with the status and if it fails, remove the claim to the pack.